libexec/rtld-elf/aarch64/rtld_start.S

```diff
@@ -185,7 +155,12 @@ ENTRY(C18N_SYM(_rtld_bind_start))
 	ldr	PTR(0), [PTR(16), #-PTR_WIDTH]
 
 	/* Call into rtld */
-	bl	_rtld_bind
+	ldr	PTR(10), rtld_bind_ptr
```

Comment:
This is being a bit evil on CHERI, it's doing a PCC-relative load of the global. Should really be loading from the GOT:
```
adrp PTR(10), :got:rtld_bind_ptr
ldr PTR(10), [PTR(10), :got_lo12:rtld_bind_ptr]
```
(or whatever the precise assembly is, if it's not that)

Reply:
(ditto for tls_get_addr_common_ptr)